tests/misc/tunsignedcomp.nim

```diff
@@ -1,6 +1,6 @@
 discard """
   output: ''''''
-  disabled: "true"
+    disabled: "true"
```

Comment:
Created around 2015-01
Disabled around 2015-01, in a commit that says "cleaned up unsigned tests"

Reply:
This doesn't work?

Pretty sure we have similar tests in the spec

Reply:
This seems like basic basic functionality, so i would expect there to be similar or redundant tests .
For completeness, the log:
```
FAIL: tests/misc/tunsignedcomp.nim c                               ( 0.45 sec)
Test "tests\misc\tunsignedcomp.nim" in category "misc"
Failure: reNimcCrash
debugInfo:  compiler exit code was 0 but some Error's were found.
$ C:\Users\Mateus\Desktop\workspace\nimskull\bin\nim.exe c --hints:on -d:testing --clearNimblePath --nimblePath:build/deps/pkgs   --nimCache:nimcache\
tests\misc\tunsignedcomp.nim_4a8a08f09d37b73795649038408b5f33  tests\misc\tunsignedcomp.nim
Hint: used config file 'C:\Users\Mateus\Desktop\workspace\nimskull\config\nim.cfg' [Conf]
Hint: used config file 'C:\Users\Mateus\Desktop\workspace\nimskull\config\config.nims' [Conf]
Hint: used config file 'C:\Users\Mateus\Desktop\workspace\nimskull\tests\config.nims' [Conf]
tunsignedcomp.nim(13, 16) Error: type mismatch: got <uint8, int8>
but expected one of:
proc `<`(x, y: uint): bool
  first type mismatch at position: 2
  required type for y: uint
  but expression '20'i8' is of type: int8
proc `<`(x, y: uint16): bool
  first type mismatch at position: 2
  required type for y: uint16
  but expression '20'i8' is of type: int8
proc `<`(x, y: uint32): bool
  first type mismatch at position: 2
  required type for y: uint32
  but expression '20'i8' is of type: int8
proc `<`(x, y: uint64): bool
  first type mismatch at position: 2
  required type for y: uint64
  but expression '20'i8' is of type: int8
proc `<`(x, y: uint8): bool
  first type mismatch at position: 2
  required type for y: uint8
  but expression '20'i8' is of type: int8
16 other mismatching symbols have been suppressed; compile with --showAllMismatches:on to see them

expression: 10'u8 < 20'i8
```

Reply:
yeah, seriously... this test just doesn't make sense, an easy fix would be to remove this particular failing line because I"m not sure if supporting that comparison even makes sense.